crates/op-rbuilder/src/tests/vanilla/smoke.rs

```diff
@@ -69,6 +70,7 @@ async fn chain_produces_blocks() -> eyre::Result<()> {
 /// Ensures that payloads are generated correctly even when the builder is busy
 /// with other requests, such as fcu or getPayload.
 #[tokio::test]
+#[cfg(feature = "integration_tests")]
```

Comment:
I'd recommend putting those conditional compilation attributes at the module inclusion level rather than decorating every test case.